kafka-elastic/src/test/java/com/baeldung/kafka/service/ElasticSearchServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.kafka.service;
+
+import com.baeldung.kafka.entity.NotificationEntity;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+
+import static org.mockito.Mockito.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.MockitoAnnotations;
+
+class ElasticSearchServiceTest {
+
+    @Mock
+    private ElasticsearchOperations elasticsearchOperations;
+
+    @InjectMocks
+    private ElasticsearchService elasticsearchService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
```

Comment:
In JUnit5, we use `@ExtendWith(MockitoExtension.class)` to avoid this boilerplate (same elsewhere)

Reply:
incoprated